fetch-runtime/package.json

```diff
@@ -5,8 +5,14 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"No test specified\"",
+    "lint": "eslint -c ../common/eslintrc.js .",
```

Comment:
I think we should convert `fetch-runtime` to typescript at some point because the `eslint` is similar, but different in places from `tslint`.